src/core.c/Mu.pm6

```diff
@@ -1026,13 +1026,15 @@ my class Mu { # declared in BOOTSTRAP
 
     method dispatch:<!>(Mu \SELF: \name, Mu \type, |c) is raw {
         my $meth := type.^find_private_method(name);
+        my $class := nqp::getlexcaller('$?CLASS');
```

Comment:
Should this be inlined so there is no additional cost for the first ternary condition?

Reply:
It should. An overlooked remnant of a debug print which followed the statement. Thanx!